src/services/railgun/wallets/wallets.ts

```diff
@@ -92,9 +97,15 @@ const loadExistingWallet = async (
 const createWallet = async (
   encryptionKey: string,
   mnemonic: string,
+  creationBlockNumbers: Optional<number[][]>,
```

Comment:
We should add some middleware here, as this `number[][]` type is confusing and kind of nasty.

So essentially this function should take the same type as FRP: `Optional<MapType<number>>`, map based on NetworkName. Then we create a quickstart function to convert to `number[][]`

Reply:
Okay sounds good, I was doing that conversion in FRP, but yeah makes sense to do it here instead